08-mmc-upload/mmc-upload.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
+
+# Source FPGA mapping
+. ${SCRIPTPATH}/../misc/crate-fpga-mapping.sh
+
+MMC_BINARIES_DIR_REL=binaries
+MMC_BINARIES_DIR=${SCRIPTPATH}/${MMC_BINARIES_DIR_REL}
+
+OPENMMC_VERSION="v1.6.0"
```

Comment:
I think it would be better to not hard-code the version here, but receive this as an argument, and deal with the idiosyncrasies of the URL for version 1.5.0.